apps/content/admin.py

```
@@ -450,18 +456,213 @@ class RecitationSurahTrackAdmin(admin.ModelAdmin):
+                            obj = RecitationSurahTrack.objects.create(
+                                asset=asset,
+                                surah_number=surah_number,
+                                audio_file=f,
+                            )
+                            try:
+                                if obj.audio_file and getattr(obj.audio_file, "name", None):
+                                    uploaded_file_names.append(obj.audio_file.name)
+                            except Exception:
+                                pass
+                            created += 1
+                except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)